### ISSUE TITLE
State of the Repo

### ISSUE THREAD

**Issue by eitanlees** [altair-viz/altair_notebooks#23]
I feel this repo has fallen behind. I think it could be a useful tool for reference as well as a place for explanations of more detailed examples such as `09-Measles.ipynb`. 

Seeing the steps involved in building more complex visualizations is a key insight which I feel is sometimes missing in the main documentation. I think the [Marginal Scatterplot](https://altair-viz.github.io/gallery/scatter_marginal_hist.html), [Wheat and Wages](https://altair-viz.github.io/gallery/wheat_wages.html), and [Seattle Weather](https://altair-viz.github.io/gallery/seattle_weather_interactive.html) examples are candidates which would benefit from showing in more detail how the components are built. 

In any case I think updating the current notebooks to `v 4.0` is important. 

I would love to help with the process of updating and maintaining the `altair_notebooks` project but I wanted to check in first to gauge the communities interest. Are there any changes you would like to see?

**Comment by jakevdp** [comment]
Yes, updating this would be great! The repo has definitely been neglected.

**Comment by eitanlees** [comment]
Hey @joelostblom! I think turning this repo into a Jupyter Book would be a great idea. It's funny, after posting this issue I never got around to actually fixing things up. So it goes ...

**Comment by eitanlees** [comment]
PRs are welcome :slightly_smiling_face:

**Comment by joelostblom** [comment]
@eitanlees Ok! I will see if I can find some time this or next weekend to do it! Btw, when the material is updated, it might be a good idea to combine it with the Pycon 2018 material into one updated learning resources? I only glanced at the content but it seems somewhat similar and I am thinking it will be easier for learners if there is either a clear distinction between the two or if they are merged. I know time is an issue, but what do you think about this conceptually?

**Comment by eitanlees** [comment]
Absolutely. Most of these notebooks are just enumerations of different chart types. Almost like a reference. I think integrating them into the Pycon 2018 material makes sense. Maybe under a new repo? or just fold them into the Pycon 2018 repo? I don't know ...

What I thought was interesting about this repo was notebooks like `09-Measles.ipynb` in which a visualization is built up from scratch showing the iteration and fine-tuning. I found seeing this process was illuminating. I thought a collection of case studies like that would be useful.

**Comment by eitanlees** [comment]
(PS I recently had my first child! Exciting! As such, time is very much an issue :sweat_smile:   
 I do very much miss working on Altair and when the time is right I look forward to contributing again.)
